base_tier_validation/tests/tier_validation_tester.py

```diff
@@ -95,9 +96,12 @@ def _compute_state(self):
                 rec.state = "draft"
 
     def action_confirm(self):
+        if self.need_validation:
```

Comment:
Unless I am mistaken this is just sidestepping this module's verification functionality